utils/jera_project_setup/__brick__/lib/support/utils/helpers/force_update.dart

```diff
@@ -0,0 +1,54 @@
+import '../../../features/force_update/models/app_info.dart';
+import '../../../features/force_update/models/minimum_version.dart';
+import '../../../features/force_update/use_case/get_app_info_use_case.dart';
+import '../../../features/force_update/use_case/get_minimum_version_use_case.dart';
+import '../../../router/mobile_router.dart';
+
+/// Classe que gerencia e avalia se o app deve ser atualizado ou não.
+///
+/// Inicia a partir da função `verifyVersion`, na qual pega essa informação do servidor,
+/// e verifica se a versão corrente nos aplicativos é a versão minima suportada pelo servidor,
+/// caso não seja redireciona o app para a tela de atualização do app.
+///
+/// ***Para utilizar corretamente esse modulo certifique-se todas as dependências necessárias estão no seu projeto.***
+class ForceUpdate {
+  static final ForceUpdate instance = ForceUpdate._();
+
+  // Private properties
```

Comment:
isso não faz mais sentido vir do service locator ?  usando o required this.getAppInfoUseCase, usei no agile a brick e senti falta disso sendo injetado ja